.travis.yml

```diff
@@ -1,12 +1,11 @@
 language: node_js
 dist: trusty
 before_script:
-  - npm install -g npm@5.2.0 # https://github.com/npm/npm/issues/17858
-  - npm install
-  - polyserve -p 8000 &
```

Comment:
Should it also be changed from `polyserve` to `serve` in `package.json` and `docs/08-_Development.md`?